Feedback bubble_sort solution

thsvr opened the pull request:
Hi,

can you please check our code and provide us feedback?

Thank you in advance :)

Regards,

Thaís and Sofi

SofiBretz committed:
first assignment done :)

thsvr committed:
second method

SofiBretz committed:
assignment 1 fixed

thsvr committed:
second method

SofiBretz committed:
linters in process

SofiBretz committed:
linters in process 1.2

SofiBretz committed:
linters in process 1.3

thsvr committed:
fix method 2

SofiBretz committed:
fix bug

SofiBretz committed:
fixed true and false on 1st and 2nd assignments

thsvr committed:
rubocop fixed

thsvr committed:
rubocop fixed

samgaco commented on bubble_sort.rb:
```diff
@@ -0,0 +1,46 @@
+
+# test
+puts bubble_sort_by(%w[hi hello hey]) { |left, right| left.length - right.length }
+# output ["hi", "hey", "hello"]
```
better not to have the tests written here for the production code

samgaco commented on bubble_sort.rb:
```diff
@@ -0,0 +1,46 @@
+y = [4, 3, 78, 2, 0, 2]
+puts bubble_sort(y)
+# output [0,2,2,3,4,78]
+
```
:warning:

samgaco reviewed (changes requested):
Hi team :high_brightness: 

Everything working fine, before the approval let's improve the read me file.

* It should be more descriptive, add your contact details, the purpose of the project (what's in there?) 
, technologies used, etc. Find some inspiration [here](https://github.com/othneildrew/Best-README-Template) or if you are feeling motivated and want to learn more about it you have this free course [here](https://www.udacity.com/course/writing-readmes--ud777).

--------------------------

For your next projects :arrow_down: 

* Choose a better name for the name of your feature branch :D 
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

--------------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

SofiBretz committed:
code review advices fixed

thsvr committed:
add readme content

thsvr commented:
Hi,

As required by Samuel Garcia, we added more information on our readme (added our contact details, the purpose of the project, live version, and technologies used). Besides that, we deleted the comments, as he required.

Can you please check if our project is able to be approved and give us feedback?

Thank you in advance :)

Thaís and Sofi

mcrd25 reviewed (approved):
# Approved. 🎊 🎉 😄 

Hi @thsvr and @SofiBretz 👋,

You guys seem to have a great sense of humor! Great job implementing the Bubble Sort! It looks great 💯 🥇 

I see no need for further review but please remember for next project to try and put a brief summary description in your pull request of all the changes you've made.
For example, I have an app that I set up and I added user model, comment model, post models, and related associations and migrations. I would mention this in my pull request description
```markdown
# Summary of changes made
- Generated User, Comment and Post Models
- Created Associations between User, Comment and Post Models
- Whatever other change you'd have made
```

Again, great job guys 🙂 


***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)
